### DISCUSSION TITLE
originalAnnouncements.toList() 获取的公告一直为0

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#106] [打扰了]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

群里面有2个公告 但是通过originalAnnouncements.toList() 获取的数量一直为0

**Comment by ForteScarlet** [comment]
与预期不符、疑似bug的内容请反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?template=bug-report.yml&assignees=ForteScarlet&labels=bug%3F)
